CLAUDE.md

```
@@ -6,48 +6,83 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
-ADMIN_PASSWORD_ADMIN=<管理者用パスワード>
-PUBLIC_R2_BASE=https://images.akyodex.com
+ADMIN_PASSWORD_OWNER=<your_secure_owner_password>
+ADMIN_PASSWORD_ADMIN=<your_secure_admin_password>
+NEXT_PUBLIC_SITE_URL=https://akyodex.com
+NEXT_PUBLIC_R2_BASE=https://images.akyodex.com
 ```
 
 ## アーキテクチャ
 
+### ディレクトリ構造（ルート配置）
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)